corehq/apps/app_manager/static/app_manager/js/visit_scheduler.js

```diff
@@ -403,28 +403,6 @@ hqDefine('app_manager/js/visit_scheduler', function () {
     };
 });
 
-// Verbatim from http://www.knockmeout.net/2011/05/dragging-dropping-and-sorting-with.html
-//connect items with observableArrays
-ko.bindingHandlers.sortableList = {
```

Comment:
This was being used instead of https://github.com/dimagi/commcare-hq/blob/je/sortable-caseaction/corehq/apps/hqwebapp/static/hqwebapp/js/knockout_bindings.ko.js#L666 but both were included in the page

Reply:
Odd, looks like we have a third copy of this in screen_config.js, almost but not quite identical: https://github.com/dimagi/commcare-hq/blob/627fe5dbce10878433176dcd3a121a64c30fee39/corehq/apps/app_manager/static/app_manager/js/details/screen_config.js#L1235

Reply:
I decided not to remove that. It is for hte case list's sorting section and it contains a `notifyButton` at the end that is what causes the save button to change. I think making the save button consistent on that page is out of scope at the moment

Reply:
I was curious, so I played around with this. Commenting out the `sortableList` binding in `screen_config.js` doesn't have any effect on the case list config. It falls back to the `sortableList` defined in `knockout_bindings.ko.js`, and the save button still works properly (I didn't dig into what's controlling the save button, just noted that it worked).

Reply:
Not sure how I tested this before. it appears its working now https://github.com/dimagi/commcare-hq/pull/23689/commits/49f57cad281e70326831fdc70442394b1e8f74dd